### PR TITLE
Add taskcard frame - TD0008-Implement-the-taskcard-session-holder

### PR DESCRIPTION
**Pull request by nguyentien290401**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,11 +1,10 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
-import { SharedModule } from './shared/shared.module';
```
Comment (thanhdat369) [inline src/app/app-routing.module.ts:3]: Why do you remove it?

File: src/app/app.module.ts [hunk 2]
```diff
@@ -3,21 +3,12 @@ import { BrowserModule } from '@angular/platform-browser';
...
 import { AppComponent } from './app.component';
-import { PopupFormComponent } from './shared-components/popup-form/popup-form.component';
-import { DeletePopupComponent } from './components/delete-popup/delete-popup.component';
-import { UpdatePopupComponent } from './components/update-popup/update-popup.component';
-import { CreatePopupComponent } from './components/create-popup/create-popup.component';
 import { SharedModule } from './shared/shared.module';
+import { MainModule } from './components/main.module';
 
 @NgModule({
-  declarations: [
```
Comment (thanhdat369) [inline src/app/app.module.ts:13]: please don't remove them if it's not necessary

Comment (nguyentien290401) [reply src/app/app.module.ts:13]: Config the Shared folder location

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (thanhdat369) [reply src/app/app-routing.module.ts:3]: <3

**Review by thanhdat369 — APPROVED** [review]

**Pull request merged**